src/library/masq.js

```diff
@@ -9,14 +9,30 @@ import { isUsernameAlreadyTaken } from './utils'
 const { encrypt, decrypt, importKey, exportKey, genAESKey, genEncryptedMasterKey, decryptMasterKey } = common.crypto
 const { dbReady, createPromisifiedHyperDB, put, get } = common.utils
 
-const { ERRORS, MasqError } = common.errors
+const { ERRORS, MasqError, checkObject } = common.errors
 
 const HUB_URLS = process.env.REACT_APP_SIGNALHUB_URLS.split(',')
 
 const swarmOpts = process.env.NODE_ENV === 'test'
   ? { wrtc: require('wrtc') }
   : {}
 
+const requiredParametersDevice = [
+  'name'
+]
+
+const requiredParametersApp = [
+  'name',
+  'description'
+]
+
+const requiredParametersProfile = [
+  'username',
```

Comment:
Even if image can be empty `''` we should check that it is present

Reply:
ok done